pypose/optim/solver.py

```diff
@@ -327,4 +327,7 @@ def forward(self, A: Tensor, b: Tensor, x: Optional[Tensor]=None,
             r -= alpha.unsqueeze(-1)*q
             rho_prev = rho_cur
 
+        assert not torch.any(torch.isnan(x)), \
+            'Conjugate Gradient Solver Failed. Check your matrix (may not be Symmetric Positive Definite)'
```

Comment:
Line is too long, refer to point 2 in this [guideline](https://github.com/pypose/pypose/wiki/Developer-Guideline).